.github/workflows/linux.yml

```diff
@@ -108,3 +113,59 @@ jobs:
         if: matrix.cli == true
         run: sh validate.sh -j 2 -w ghc-${{ matrix.ghc }} -v ${{ matrix.flags }} -s cli-suite
 
+  validate-old-ghcs:
+    name: Validate old ghcs ${{ matrix.extra-ghc }}
+    runs-on: ubuntu-18.04
+    needs: validate
+    container:
+      image: phadej/ghc:8.8.4-xenial
```

Comment:
We might want to add a comment or two on why we are picking this old image. Apparently both you and I had to rediscoved the reason, if we don't write it down it might get lost again. (Admittedly I am about to forget again).